example/16_gemm_reduce/gemm_reduce_xdl_mean_squaremean_fp16.cpp

```diff
@@ -47,10 +47,12 @@ using DxsReduceOp = ck::Tuple<D0ReduceOp, D1ReduceOp>;
 
 using UnaryIdenticElementOp =
     ck::tensor_operation::element_wise::UnaryIdentic<ReduceAccDataType, ReduceAccDataType, false>;
+using UnaryDivElementOp =
```

Comment:
@qianfengz

It is so strange to use ```UnaryIdentic``` for division.

The pointwise function used by reduction operator should be refactored to use the same pointwise functions as other operations. Please produce a plan for this change